data/dictionary.txt

```diff
@@ -2230,6 +2247,7 @@ intenational->international
 intented->intended, indented,
 intepretation->interpretation
 intepretator->interpretor
+interal->internal
```

Comment:
I guess this should be a suggestion instead... could be "integral", particularly in SW dealing with math.